doc/manual.asciidoc

```diff
@@ -619,6 +619,20 @@ rule cc
   command = cl /showIncludes -c $in /Fo$out
 ----
 
+3. (since Ninja 1.11) `deps = msvc_source_dependencies` specifies that the tool
```

Comment:
I'm not familiar with Ninja's release cycle. Is this the right release if I go into master now?

Reply:
I believe so, yes.

Reply:
Should be, yes.